src/test/java/io/vertx/cassandra/StreamingTest.java

```diff
@@ -0,0 +1,44 @@
+package io.vertx.cassandra;
+
+import io.vertx.core.Future;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.regex.Pattern;
+
+@RunWith(VertxUnitRunner.class)
+public class StreamingTest extends CassandraServiceBase {
+
+  @Test
+  public void simpleStreamingReleaseVersionSelect(TestContext context) {
```

Comment:
It would be better to have a test for a query returning at least a few rows.

Reply:
Agree. I think we should we should merge #16 first, so we will have some data for retrieving.